webapp/src/main/java/com/box/l10n/mojito/rest/repository/RepositoryWS.java

```diff
@@ -78,35 +78,19 @@ public Repository getRepositoryById(@PathVariable Long repositoryId)
   }
 
   /**
-   * Gets all undeleted repositories with @{link View.RepositorySummary}
-   *
-   * @return List of {@link Repository}s
-   */
-  @JsonView(View.RepositorySummary.class)
-  @RequestMapping(
-      value = "/api/repositories",
-      method = RequestMethod.GET,
-      produces = MediaType.APPLICATION_JSON_VALUE)
-  @ResponseStatus(HttpStatus.OK)
-  public List<Repository> getRepositories() {
-    return repositoryService.findRepositoriesIsNotDeletedOrderByName(null);
-  }
-
-  /**
-   * Gets repository matching the given name
+   * Gets repository matching the given name (optional)
    *
    * @param repositoryName To filer on the name. Can be {@code null}
    * @return List of {@link Repository}s
    */
   @JsonView(View.Repository.class)
   @RequestMapping(
       value = "/api/repositories",
-      params = "name",
       method = RequestMethod.GET,
       produces = MediaType.APPLICATION_JSON_VALUE)
   @ResponseStatus(HttpStatus.OK)
   public List<Repository> getRepositories(
-      @RequestParam(value = "name", required = true) String repositoryName) {
```

Comment:
Two methods with the same path and HTTP method (GET) is not supported by Open API 3